### PR TITLE
Deprecations for 6.10

### PR DESCRIPTION
**Pull request by ryandeussing**
This is a re-work of https://github.com/theforeman/foreman-documentation/pull/600 which hit rebase issues related to files that had been deleted from master

https://issues.redhat.com/browse/SAT-4179

Cherry-pick into:

* [x] Foreman 2.5 (Satellite 6.10)
* [ ] Foreman 2.4
* [ ] Foreman 2.3 (Satellite 6.9)
* [ ] Foreman 2.1 (Satellite 6.8)

### CODE REVIEW

**Comment by ryandeussing** [comment]
@ianf77 as mentioned, I was unable to resolve the conflict introduced by multiple commits referencing a deleted upstream file  (along with other files) so I took it upon myself to recreate your changes on a new branch. Not the most elegant solution but the git-foo required to stay on #600 was beyond me.

**Comment by lzap** [comment]
Hello and thank you for the contribution!

Usually in Satellite docs the deprecations are just listed in the release notes. Is this removal of already-deprecated stuff from 6.9?

What is the status of this PR?

**Comment by ianf77** [comment]
This PR is current, it has removal of a few of the items deprecated at 6.9 and not present in 6.10 that were not covered by other PRs. It also has notes indicating items deprrecated at 6.10 that will be removed at 7.0. It

**Comment by ryandeussing** [comment]
@spetrosi @melcorr requested changes have been made, can this PR be merged? 

Thank you.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 1]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
 
 To create the plan, complete the following steps:
 
-. In the {ProjectWebUI}, navigate to *Insights* > *Inventory*, and select the hosts that you want to include in an Insights plan.
-. From the *Actions* list, select *Create a new Plan/Playbook*.
-. From the Plan/Playbook Builder window, select *Create new plan* and enter a name for the plan.
-. Select whether you want the rule to apply to a specific system, group, or all systems.
-. Select one or more rules that you want to add to the plan.
+In the {Project} web UI, navigate to *Configure* *>* *Insights*.
```
Comment (maximiliankolb) [inline guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:10]:
```suggestion
In the {ProjectWebUI}, navigate to *Configure* *>* *Insights*.
```

File: guides/doc-Planning_Guide/topics/Glossary.adoc [hunk 2]
```diff
@@ -128,9 +128,11 @@ In relationship to a host, erratum is *applicable* if it updates a package insta
 
 
 [[varl-Glossary_of_Terms-External_Node_Classifier]]
-*External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
+*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
 {ProjectName} acts as an External Node Classifier to Puppet servers in a {Project} deployment.
-
++
+Note that the External Node Classifier will be removed in the next {Project} version.
++
```
Comment (maximiliankolb) [inline guides/doc-Planning_Guide/topics/Glossary.adoc:135]:
```suggestion
```

File: guides/doc-Planning_Guide/topics/Glossary.adoc [hunk 3]
```diff
@@ -156,7 +158,11 @@ To see the full list of services, enter the `{foreman-maintain} service list` co
 
 [[varl-Glossary_of_Terms-Foreman_Hooks]]
 *Foreman Hook*:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
-
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.\
++
```
Comment (maximiliankolb) [inline guides/doc-Planning_Guide/topics/Glossary.adoc:164]:
```suggestion
```

File: guides/doc-Planning_Guide/topics/Glossary.adoc [hunk 4]
```diff
@@ -156,7 +158,11 @@ To see the full list of services, enter the `{foreman-maintain} service list` co
 
 [[varl-Glossary_of_Terms-Foreman_Hooks]]
 *Foreman Hook*:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
-
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.\
```
Comment (maximiliankolb) [inline guides/doc-Planning_Guide/topics/Glossary.adoc:163]:
```suggestion
Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
```

**Review by maximiliankolb — COMMENTED** [review]
A few suggestions, but rest LGTM.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc (hunk 1 above)
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:10]: Fixed

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 2 above)
Comment (ianf77) [reply guides/doc-Planning_Guide/topics/Glossary.adoc:135]: Fixed

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 4 above)
Comment (ianf77) [reply guides/doc-Planning_Guide/topics/Glossary.adoc:163]: Fixed.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc [hunk 5]
```diff
@@ -90,7 +90,6 @@ This includes Content Views, Activation Keys, and Life Cycle Environments.
 | *Hosts*  | Provides host inventory and provisioning configuration tools.
 | *Configure*  | Provides general configuration tools and data including Host Groups and Puppet data.
 | *Infrastructure*  | Provides tools on configuring how {ProjectX} interacts with the environment.
-| *Insights*  | Provides Red{nbsp}Hat Insights management tools.
```
Comment (lzap) [inline guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: What happened to RH Insights plugin? @ares I only see it mentioned in the issue.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 6]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
...
+. In the *Job Invocation* page, do not change the contents of precompleted fields.
+. Optionally, for more advanced configuration of the Remote Execution Job you can click *Show Advanced Fields*.
+. Select the *Type of query* you require.
+. Select the *Schedule* you require.
+. Click *Submit*.
 
-In the Jobs window, you can view the progress of your plan as the playbook runs.
-
-You can view the Insights plan by navigating to *Insights* > *Planner*.
+In the Jobs window, you can view the progress of your plan.
```
Comment (lzap) [inline guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:25]: Can someone confirm this @ares ?

File: guides/doc-Planning_Guide/topics/Glossary.adoc [hunk 7]
```diff
@@ -128,9 +128,10 @@ In relationship to a host, erratum is *applicable* if it updates a package insta
 
 
 [[varl-Glossary_of_Terms-External_Node_Classifier]]
-*External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
+*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
```
Comment (lzap) [inline guides/doc-Planning_Guide/topics/Glossary.adoc:131]: Just construct read weird, perhaps a component?

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 3 above)
Comment (lzap) [reply guides/doc-Planning_Guide/topics/Glossary.adoc:163]: We should state that webhooks is an alternative now, @ares ?

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 7 above)
Comment (ryandeussing) [reply guides/doc-Planning_Guide/topics/Glossary.adoc:131]: @ian your call

**Review by maximiliankolb — CHANGES_REQUESTED** [review]
If it's just "Deprecations for 6.10", then there's no need to cherry-pick in for Foreman 2.4 IMHO.

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 7 above)
Comment (ianf77) [reply guides/doc-Planning_Guide/topics/Glossary.adoc:131]: Comments made on PR#600, the precursor of this PR 
 @ekohl ekohl on Jul 22

I've been trying to convince people that Foreman should adopt the term ENC and not consider it a Puppet-only concept. However, this appears to fall on deaf ears. Maybe at this point I'm just ranting.

melcorr on Jul 23 Member 
And how would that look?
Suggested change
*External Node Classifier*:: A Puppet construct that provides additional data for a Puppet server to use when configuring hosts.
*External Node Classifier*:: A construct that provides additional data for a server to use when configuring hosts.
Like this?

That's why it was changed.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 8]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
...
-. In the {ProjectWebUI}, navigate to *Insights* > *Inventory*, and select the hosts that you want to include in an Insights plan.
-. From the *Actions* list, select *Create a new Plan/Playbook*.
-. From the Plan/Playbook Builder window, select *Create new plan* and enter a name for the plan.
-. Select whether you want the rule to apply to a specific system, group, or all systems.
-. Select one or more rules that you want to add to the plan.
+In the {ProjectWebUI}, navigate to *Configure* *>* *Insights*.
+. On the *Red Hat Insights* page, select the number of recommendations that you want to include in an Insights plan.
++
+You can only select the recommendations that have an associated playbook.
++
```
[the remark this answers is not in the document]
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:14]: Fixed.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 9]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
 
 To create the plan, complete the following steps:
 
-. In the {ProjectWebUI}, navigate to *Insights* > *Inventory*, and select the hosts that you want to include in an Insights plan.
-. From the *Actions* list, select *Create a new Plan/Playbook*.
-. From the Plan/Playbook Builder window, select *Create new plan* and enter a name for the plan.
-. Select whether you want the rule to apply to a specific system, group, or all systems.
-. Select one or more rules that you want to add to the plan.
+In the {ProjectWebUI}, navigate to *Configure* *>* *Insights*.
```
[the remark this answers is not in the document]
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:10]: Fixed.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 10]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
...
+You can only select the recommendations that have an associated playbook.
++
+. Click *Remediate*.
+. In the *Remediation Summary* window, you can select the *Resolutions* to apply.
 Use the *Filter* field to search for specific keywords.
-. Click *Save*.
-. In the Planner window, select *Run Playbook*.
+. Click *Remediate*.
+. In the *Job Invocation* page, do not change the contents of precompleted fields.
+. Optionally, for more advanced configuration of the Remote Execution Job you can click *Show Advanced Fields*.
```
[the remark this answers is not in the document]
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:20]: Fixed

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc [hunk 11]
```diff
@@ -7,15 +7,19 @@ With {ProjectX}, you can create a Red{nbsp}Hat Insights remediation plan and run
 
 To create the plan, complete the following steps:
 
-. In the {ProjectWebUI}, navigate to *Insights* > *Inventory*, and select the hosts that you want to include in an Insights plan.
-. From the *Actions* list, select *Create a new Plan/Playbook*.
-. From the Plan/Playbook Builder window, select *Create new plan* and enter a name for the plan.
-. Select whether you want the rule to apply to a specific system, group, or all systems.
-. Select one or more rules that you want to add to the plan.
+In the {ProjectWebUI}, navigate to *Configure* *>* *Insights*.
+. On the *Red Hat Insights* page, select the number of recommendations that you want to include in an Insights plan.
```
[the remark this answers is not in the document]
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:11]: Fixed.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc (hunk 5 above)
Comment (ianf77) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: RH Insights plugin is removed at 6.10.

File: guides/doc-Planning_Guide/topics/Glossary.adoc (hunk 3 above)
Comment (ianf77) [reply guides/doc-Planning_Guide/topics/Glossary.adoc]: Fixed

**Comment by ryandeussing** [comment]
@ares @lzap what can we do to move this across the finish line? ty

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc (hunk 5 above)
Comment (ares) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: The old plugin, it's replaced by rh cloud plugin

File: guides/common/modules/proc_adding-rhv-connection.adoc [hunk 12]
```diff
@@ -34,10 +33,8 @@ Alternatively, if you leave the field blank, a self-signed certificate is genera
 --name "__My_{oVirtShort}__" --provider "Ovirt" \
 --description "{oVirtShort} server at _{ovirt-example-com}_" \
 --url "_https://{ovirt-example-com}/ovirt-engine/api_" \
---use-v4 "false" --user "_{Project}_User_" \
---password "_My_Password_" \
+--user "_{Project}_User_" --password "_My_Password_" \
 --locations "New York" --organizations "_My_Organization_" \
 --datacenter "_My_Datacenter_"
 ----
 +
```
Comment (spetrosi) [inline guides/common/modules/proc_adding-rhv-connection.adoc:40]: Remove this trailing line
```suggestion
```

File: guides/doc-Planning_Guide/topics/Glossary.adoc [hunk 13]
```diff
@@ -156,7 +157,11 @@ To see the full list of services, enter the `{foreman-maintain} service list` co
 
 [[varl-Glossary_of_Terms-Foreman_Hooks]]
 *Foreman Hook*:: An executable that is automatically triggered when an orchestration event occurs, such as when a host is created or when provisioning of a host has completed.
+ifdef::satellite[]
++
+Note that Foreman Hook functionality is deprecated and will be removed in the next {Project} version.
```
Comment (spetrosi) [inline guides/doc-Planning_Guide/topics/Glossary.adoc:163]: Remove this newline
```suggestion
```

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc (hunk 5 above)
Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: We need to check if the Insights tab is still in place, Marek says that it has been replaced, not removed completely

Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: The Insights tab is referred in the below changes in this PR

**Comment by spetrosi** [comment]
@ryandeussing this is not a standard practice to mark depredations throughout the docs. Usually, we only marked deprecated functionality in the Release Notes, in the Deprecated Functionality section.

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc (hunk 5 above)
Comment (ianf77) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: From SAT-4179
The following items deprecated in 6.9 should be dropped 6.10:
   Red Hat Access Insights plugin 

Comment on SAT-3192 by Mike McCune
The functionality in Satellite that was available in the top level navigation under the Insights menu options has now been directly integrated into the Hosts -> All Hosts user interface. There is no longer a separate Insights UI as we work to unify the experience of recommending and remediating actions for the hosts managed in Satellite.

File: guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc (hunk 6 above)
Comment (ianf77) [reply guides/doc-Managing_Hosts/topics/creating-an-insights-plan.adoc:25]: It was originally confirmed by Ondrej Prazak

File: guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc (hunk 5 above)
Comment (ianf77) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: @spetrosi Does this answer the questions? Can we get this merged?

Comment (spetrosi) [reply guides/doc-Administering_Red_Hat_Satellite/topics/Logging_in_to_Red_Hat_Satellite.adoc:93]: Yup, thanks for the clarification, it's clear now. I do not have the permissions to merge PRs.

**Review by spetrosi — APPROVED** [review]
lgtm

**Comment by ekohl** [comment]
Pressing the green button since @spetrosi approved it.

**Pull request merged**